.env.sample

```diff
@@ -1,3 +0,0 @@
-REACT_APP_CLIENT_ID=example
-REACT_APP_OKTA_ISSUER_URI="https://auth.lambdalabs.dev/oauth2/default"
-REACT_APP_API_URI=http://localhost:3000
```

Comment:
it seems like u need to have this file in the project root folder even when u don't use it

Reply:
I just removed the `.sample` on that file. I didn't delete it. Not sure if that makes a difference but I can add it back no problem

Reply:
yeah it seems like changing the name of a file will count like delete the one with the old name and create a new one, and that's the conflict

Reply:
yeah, you have to keep the .env.sample file in the repo because the .env file does not get uploaded, and the GitHub tests will fail without it

Reply:
I just pushed new changes. added the .env.sample back so we should be good to go now